exc1/src/main/java/dgodek/company/TooManyEmployees.java

```diff
@@ -0,0 +1,10 @@
+package dgodek.company;
+
+/**
+ * Created by matematyk60 on 23.07.17.
+ */
+public class TooManyEmployees extends RuntimeException {
```

Comment:
Maybe TooManyEmployeesException, but I actually can't see anything exceptional in manager that is not having enough place in team to hire another person(unless you are following exception driven development path, then we're fine)